internal/generate/olm-catalog/csv.go

```diff
@@ -180,6 +187,14 @@ func (g *BundleGenerator) setDefaults() {
 // olmapiv1alpha1.ClusterServiceVersion instead of writing to a template.
 func (g BundleGenerator) Generate() error {
 	g.setDefaults()
+
+	csvPath := g.getCSVPath(g.OperatorName)
+
+	if (g.HasInteractivePreference && g.EnableInteractivePrompt) ||
```

Comment:
Setting `g.EnableInteractivePrompt = true` implies the caller wants an interactive prompt, so the first preference check is redundant:
```suggestion
	if g.EnableInteractivePrompt ||
```

Reply:
Since is  bool lets add a prefix to make it intuitive IsEnableInteractivePrompt